frontend/pulse3d/src/.env.production

```diff
@@ -1,4 +1,4 @@
-NEXT_PUBLIC_PULSE3D_URL=https://pulse3d.curibio-test.com
-NEXT_PUBLIC_USERS_URL=https://apiv2.curibio-test.com/users
-NEXT_PUBLIC_MANTARRAY_URL=https://apiv2.curibio-test.com/mantarray
-NEXT_PUBLIC_CLUSTER=test
+NEXT_PUBLIC_PULSE3D_URL=https://pulse3d.curibio-modl.com
+NEXT_PUBLIC_USERS_URL=https://apiv2.curibio-modl.com/users
+NEXT_PUBLIC_MANTARRAY_URL=https://apiv2.curibio-modl.com/mantarray
+NEXT_PUBLIC_CLUSTER=modl
```

Comment:
should this be in a `.env.modl` file instead of `.env.production`?

Reply:
No, the `.env.production` is used by react in the `npm run export:<env>` command so the values get updated anytime someone exports new files for an environment. The `.env.development` is used when running the FE locally with `npm run dev`